lighthouse-core/test/lib/network-request-test.js

```diff
@@ -90,16 +90,16 @@ describe('NetworkRequest', () => {
       };
     }
 
-    it('updates lrStatistics if in Lightrider', () => {
+    it('updates fetch stats and lrStatistics if in Lightrider', () => {
```

Comment:
```suggestion
    it('updates fetch timing numbers if in Lightrider', () => {
```
really this is about those, `lrStatistics` is just the logging instrument, so makes sense to focus on the timing numbers

Reply:
should maybe update the `'does not update lrStatistics when'` below to the same "does nothing if" used in the other tests, too